### DISCUSSION TITLE
duplicate

### DISCUSSION THREAD

**Discussion by karolis-eigelis** [aclap-dev/video-downloadhelper#1075] [Help and Q&A]
no

**Comment by qmarsn** [comment]
is there anyway to exclude a download link if it has already been downloaded? thanks

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

You really ought to start a new discussion.  Your comment is in no way a continuation of the topic already here.  In fact, the topic already here is empty.  You are hurting your chances of getting any real help by posting in this discussion.

When you do open a new discussion -- when, not if, when -- you need to include the `Copy information to clipboard` data that the discussion creation process prompts you for.  Don't skip it.  Don't ignore it.  You should also post some screenshots of what exactly you are looking for.  I don't understand what you are trying to ask for, so some pictures would help your cause greatly.  Pictures.  Worth way more than 1000 words.  Screenshots.  Show what you are getting & explain what you would rather get in its place.
